### PR TITLE
chore: add verve

### PR DESCRIPTION
**Pull request by ParthJadhav**
This is the link to the project: [Verve](https://github.com/ParthJadhav/verve)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.

### Additional Context

Verve is a lightweight and blazingly fast launcher for accessing and opening applications, files and documents. It serves as a replacement for popular tools such as Spotlight, Raycast, and Alfred, and is written in the Rust programming language using the Tauri Framework.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
 - [Wirefish](https://github.com/stefanodevenuto/wirefish) - Cross-platform packet sniffer and analyzer.
 - [Kanri](https://github.com/trobonox/kanri) - Cross-platform, offline-first Kanban board app with a focus on simplicity and user experience.
+- [Verve](https://github.com/ParthJadhav/verve) - launcher for accessing and opening applications, files and documents.
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [Verve](https://github.com/ParthJadhav/verve) - Launcher for accessing and opening applications, files and documents.
```

**Review by JonasKruckenberg — DISMISSED** [review]

**Comment by JonasKruckenberg** [comment]
You should also consider adding a strict CSP here https://github.com/ParthJadhav/Verve/blob/19ff2345331b9a3522321fff4400a26e0b068b88/src-tauri/tauri.conf.json#L84

**Comment by ParthJadhav** [comment]
> You should also consider adding a strict CSP here https://github.com/ParthJadhav/Verve/blob/19ff2345331b9a3522321fff4400a26e0b068b88/src-tauri/tauri.conf.json#L84

Can you point me to a resource where I can read what it does?

**Comment by JonasKruckenberg** [comment]
> > You should also consider adding a strict CSP here https://github.com/ParthJadhav/Verve/blob/19ff2345331b9a3522321fff4400a26e0b068b88/src-tauri/tauri.conf.json#L84
> 
> Can you point me to a resource where I can read what it does?

https://tauri.app/v1/api/config/#securityconfig.csp

**Comment by JonasKruckenberg** [comment]
It also appears at least the aarch64 release is broken
Screenshot 2023-01-09 at 11 28 37

**Comment by ParthJadhav** [comment]
> It also appears at least the aarch64 release is broken Screenshot 2023-01-09 at 11 28 37

It's just with apple's security policies.. Due to the fact that Verve is not signed we gotta do some stuff to get it running:

An extra step is required to run the M1 (arm64) build :

Open the terminal, type the following command, and then press the return key:
```
xattr -cr "<installation location>.app"
```

**Comment by ParthJadhav** [comment]
> > > You should also consider adding a strict CSP here https://github.com/ParthJadhav/Verve/blob/19ff2345331b9a3522321fff4400a26e0b068b88/src-tauri/tauri.conf.json#L84
> > 
> > 
> > Can you point me to a resource where I can read what it does?
> 
> https://tauri.app/v1/api/config/#securityconfig.csp

Added the CSP properties:
https://github.com/ParthJadhav/Verve/commit/d0f00a541360471e155b2d9f3b9af31971c0a678

**Review by FabianLars — APPROVED** [review]

**Pull request merged**